models/benchmarks/benchmark_model.py

```diff
@@ -39,5 +40,18 @@ def get_plugin(self):
         """Returns the plugin to parse the results"""
         pass
 
+    def parse_run_args(self, run_args):
```

Comment:
Unfortunately, benchmark arguments are very different from each other, so we should not attempt to map all possible choices, or we'll end up with a nightmare to mix and match. Arguments should be a string, passed down from command line. Internal arguments (for different multiple runs) are specific to each benchmark and should not be here.

Reply:
Individual syntax is different, yes, but their goal is the same (complexity of the problem, size...)
Logic here would be to have standard flags that would then be up to each benchmark's model to be translated into the right syntax.